components/language-picker/common/find-language/languageTagUtils.ts

```diff
@@ -0,0 +1,34 @@
+import { getShortestSufficientLangtag } from "./getShortestSufficientLangtag";
+
+export function createTag({
+  languageCode,
+  scriptCode,
+  regionCode,
+  dialectCode,
+}: {
+  languageCode?: string;
+  scriptCode?: string;
+  regionCode?: string;
+  dialectCode?: string;
+}) {
+  if (!languageCode) {
```

Comment:
Why is the script code ignored in this `!languageCode` case?

Reply:
We have fixed this, thank you